exe/deploy_collection_manifest

```diff
@@ -23,16 +23,14 @@ end
 new_collection_manifest = ARGV.shift
 ingest_manifest = ARGV.shift
 sfs = ARGV.shift
+# TODO:  archives_list is no longer used, so this should be removed. It's being kept for the moment to ensure
+# that the script doesn't break because of the missing ARGV.shift.
 archives_list = ARGV.shift || 'archival01,archival02,archival03,archival04,archival05,archival06,archival07'
```

Comment:
I don't think we need to get archives_list from argument anymore and line 28 can be removed.